sourcegraph/code-intel-extensions-bundles
# bundle default extensions

Adds default extension bundles.

Note: extension manifests and bundles are generated and don't need to be reviewed. The main changes introduced by this PR are in `index.js`, `bundles/publish.js`, and `README` files.

## Test plan

- set up a Sourcegraph instance (e.g. [Docker single container](https://docs.sourcegraph.com/admin/deploy/docker-single-container))
- [disable remote extension registry](https://docs.sourcegraph.com/admin/extensions#use-extensions-from-sourcegraph-com-or-disable-remote-extensions)
- add a valid license key to your instance to be able to use the local extension registry (e.g. copy it from the dogfood instance)
- clone this repo
- run `npm run publish`
- ensure that extensions are published to your instance local extension registry
- ensure that code intel and default extensions work on your instance
